commercetools/resource_discount_code_test.go

```diff
@@ -119,12 +107,6 @@ func TestAccDiscountCodeCreate_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"commercetools_discount_code.standard", "max_applications", "0",
 					),
-					resource.TestCheckResourceAttr(
-						"commercetools_discount_code.standard", "cart_discounts.0", "cart-discount-id-0",
-					),
-					resource.TestCheckNoResourceAttr(
```

Comment:
This check can be added back, the previous one can't because the uuid will always change.